### PR TITLE
Add Curse of the Dragonborn - REDUX

### LINKED ISSUE #2873: Curse of the Dragonborn REDUX special load order instructions
https://www.nexusmods.com/skyrimspecialedition/mods/104477?tab=description

Putting this here because I have no idea how to implement something like this, or if there's even an easy way to do it. LOOT places this mod near the bottom of the load order, but the description page says that the mod should be loaded before any vampire and/or werewolf overhauls are loaded. It's also been noted in the comments that this mod will require minor conflict resolution with any mods that modify the ring of hircine, though I'm not 100% sure that's the sort of thing LOOT would push a notification for.

### PR DESCRIPTION
**Pull request by SkyLover264**
Added the mod in question to Early Loaders so it'll load before vampire and werewolf overhauls, as per the page instructions.

### CODE REVIEW

**Comment by SkyLover264** [comment]
Did I code this right? It's been a while since I've proposed edits to the LOOT masterlist.

File: masterlist.yaml [hunk 1]
```diff
@@ -9121,6 +9121,10 @@ plugins:
   - name: 'Crossbow Integration - USSEP.esp'
     tag: [ Text ]
 
+  - name: 'Curse of the Dragonborn - REDUX'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/104477/' ]
```
Comment (sibir-ine) [inline masterlist.yaml]: A plugin's `name` node value needs to be the plugin's filename. You can use that name for the location though:

```suggestion
  - name: 'OPHybrid.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/104477/'
        name: 'Curse of the Dragonborn - REDUX'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Comment by SkyLover264** [comment]
What now? I corrected the entry to reflect how it's supposed to be done, and now it's failing the checks, when the incorrect version passed without issue. I'm so confused.

File: masterlist.yaml [hunk 2]
```diff
@@ -9121,6 +9121,11 @@ plugins:
   - name: 'Crossbow Integration - USSEP.esp'
     tag: [ Text ]
 
+  - name: 'OPHybrid.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/104477/'
+	name: 'Curse of the Dragonborn - REDUX'
+
```
Comment (sibir-ine) [inline masterlist.yaml]: You've used a tab character here which causes the CI to fail, indentation should only be done with spaces. You also accidentally removed the group assignment & added some trailing spaces. GitHub should give you the ability to commit my suggestion in the website without having to do it manually on your own:
```suggestion
        name: 'Curse of the Dragonborn - REDUX'
    group: *earlyLoadersGroup

```

Comment (SkyLover264) [reply masterlist.yaml]: It wasn't letting me commit your suggestions, so I did them manually, including this one. It kept saying "This diff has been updated. Refresh and try again." Either way, I replaced the tabs with spaces, and it passed all checks. Do tabs have a special meaning in this programming language or something? Every other language I've used either didn't care, or actively expected tabs to be used.

Comment (sibir-ine) [reply masterlist.yaml]: Ah, GH does that sometimes, still not sure why. Re tab characters: they're illegal in YAML, which the masterlists use, due to inconsistent implementation across editors IIRC.

File: masterlist.yaml [hunk 3]
```diff
@@ -9121,6 +9121,12 @@ plugins:
   - name: 'Crossbow Integration - USSEP.esp'
     tag: [ Text ]
 
+  - name: 'OPHybrid.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/104477/'
+        name: 'Curse of the Dragonborn - REDUX'
+    group: *earlyLoadersGroup
+    
   - name: 'CutContentRestoration.esp'
```
Comment (sibir-ine) [inline masterlist.yaml]: There are still some trailing spaces here:
```suggestion
    group: *earlyLoadersGroup

  - name: 'CutContentRestoration.esp'
```

Comment (SkyLover264) [reply masterlist.yaml]: Whoops. Taken care of.

**Review by sibir-ine — APPROVED** [review]
Thanks!

File: masterlist.yaml [hunk 4]
```diff
@@ -9121,6 +9121,12 @@ plugins:
   - name: 'Crossbow Integration - USSEP.esp'
     tag: [ Text ]
 
+  - name: 'OPHybrid.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/104477/'
+        name: 'Curse of the Dragonborn - REDUX'
+    group: *earlyLoadersGroup
+
```
Comment (pStyl3) [inline masterlist.yaml]: Now that the plugin is correctly named `OPHybrid.esp`, the complete entry should be sorted alphabetically within the `Gameplay` category. So please sort `OPHybrid.esp` inbetween `Open World Loot.esp` and `OWL Challenge Mode.esp`. After that this PR can be merged.

**Review by pStyl3 — CHANGES_REQUESTED** [review]

**Review by pStyl3 — APPROVED** [review]

**Comment by pStyl3** [comment]
Thanks!

**Pull request merged**